flask_bootstrap/templates/base/utils.html

```diff
@@ -10,14 +10,18 @@
 {% endmacro %}
 
 
-{% macro render_icon(name, size=config.BOOTSTRAP_ICON_SIZE, color=config.BOOTSTRAP_ICON_COLOR, title=None, desc=None) %}
+{% macro render_icon(name, size=config.BOOTSTRAP_ICON_SIZE, color=config.BOOTSTRAP_ICON_COLOR, title=None, desc=None, font=False) %}
 {% set bootstrap_colors = ['primary', 'secondary', 'success', 'danger', 'warning', 'info', 'light', 'dark', 'muted'] %}
+{%- if font == true -%}
```

Comment:
How about adding a global configuration aside from the `font` param? For example, `BOOTSTRAP_ICON_USE_FONT`. Then users don't need to set this `font` param for all the `render_icon` calls.

Reply:
Please also update docs for this config variable.

Reply:
I use the two forms of icon rendering interchangeably, so I don't want to force the user to use only on of the two. Is that okay with you?

Reply:
The config defaults to `False`. The user can then choose their preferred method: either set the config to `True `or control the specific icon using the font parameter.

Reply:
Ah, now I get it, only the default value is set via global var. Will change it.

Reply:
done